objcToxBot/CallAudioTask.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CallAudioTask.h
+//  objcToxBot
+//
+//  Created by Chuong Vu on 9/4/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "TaskProtocol.h"
+
+@interface CallAudioTask : NSObject <TaskProtocol>
```

Comment:
Can you please add comment about what this task will do (you can check other tasks as example)?